src/app/components/radio/index.html

```diff
@@ -1,13 +1,268 @@
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-11.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use icon radio buttons when the icons may be unclear.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-12.png" />
+                    <div class="caption caption-success sky-text-success">Do use radio buttons with text labels when icons are not immediately obvious.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Don’t use ungrouped icon radio buttons when the choices could be grouped together on the same line.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div class="sky-margin-stacked-content-section">
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-13.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use individual icon radio buttons when they could be grouped.</div>
+                    </div>
+                    <div>
+                    <img src="~/assets/img/guidelines/radio-button/radio-button-usage-14.png" />
+                    <div class="caption caption-success sky-text-success">Do use icon radio groups when possible.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+            
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Anatomy">
+        
+            <h3>Regular radio buttons</h3>
+            <sky-docs-anatomy imagePath="~/assets/img/guidelines/radio-button/radio-button-anatomy-1.png">
+                <sky-docs-anatomy-item anatomyNumber="1">Radio glyph</sky-docs-anatomy-item>
+                <sky-docs-anatomy-item isOptional="true" anatomyNumber="2">Text label</sky-docs-anatomy-item>
+            </sky-docs-anatomy>
+            
+            <h3>Icon radio buttons</h3>
+            <sky-docs-anatomy imagePath="~/assets/img/guidelines/radio-button/radio-button-anatomy-2.png">
+                <sky-docs-anatomy-item anatomyNumber="1">Radio type</sky-docs-anatomy-item>
+                <sky-docs-anatomy-item anatomyNumber="2">Icon</sky-docs-anatomy-item>
+            </sky-docs-anatomy>
+            
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Options">
+            
```

Comment:
Should regular vs icon be called out as options or is that self-evident?

Reply:
If it's not, the use cases for each mode are covered in the Usage guidelines.

Reply:
Resolving since it seems like we opted not to treat this distinction as an option.